File: src/main/java/org/lenteam/colmen/Application.java
package org.lenteam.colmen;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * @author Anton_Solovev
 * @since 8/20/2016
 */

@SpringBootApplication
@EnableAutoConfiguration
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: src/main/kotlin/ru/ssau/tourism/initializer/Application.kt
package ru.ssau.tourism.initializer

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
 * @since 31.10.16.
 */
@SpringBootApplication
open class Application {
}

fun main(args: Array<String>) {
	SpringApplication.run(Application::class.java, *args)
}
